pkg/curatedpackages/certmanagerclient.go

```
@@ -19,10 +19,10 @@ func NewCertManagerClient(kubectl KubectlRunner, kubeConfig string) *CertManager
 }
 
 // CertManagerExists checks if cert-manager exists in any namespace in the cluster.
-func (cmc *CertManagerClient) CertManagerExists(ctx context.Context) (bool, error) {
+func (cmc *CertManagerClient) CertManagerExists(ctx context.Context) bool {
 	// Note although we passed in a namespace parameter in the kubectl command, the GetResource command will be
 	// performed in all namespaces since CRDs are not bounded by namespaces.
-	found, err := cmc.kubectl.GetResource(ctx, "crd", "certificates.cert-manager.io", cmc.kubeConfig, constants.CertManagerNamespace)
+	found, _ := cmc.kubectl.GetResource(ctx, "crd", "certificates.cert-manager.io", cmc.kubeConfig, constants.CertManagerNamespace)
```

Why are we ignoring the error?